lib/queues.js

```diff
@@ -483,7 +485,9 @@ class Queue {
       // Error or timeout: we release the job back to the queue.  Since this
       // may be a transient error condition (e.g. server down), we let it sit
       // in the queue for a while before it becomes available again.
-      const delay = msToSec(ifProduction(RELEASE_DELAY));
+      const delay = msToSec(ifProduction(getReleaseDelay(job.reserveCount || 1)));
+
+      debug('Releasing job %s:%s with delay of %ds', self.name, jobID, delay);
```

Comment:
This gives out the delay in seconds, but anyone looking at the logs, wouldn't they care more about the time, or the approximate delay in minutes?